src/woo_publications/metadata/api/tests/test_information_category_api_endpoints.py

```diff
@@ -0,0 +1,152 @@
+from django.urls import reverse
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from woo_publications.metadata.constants import InformationCategoryOrigins
+from woo_publications.metadata.tests.factories import InformationCategoryFactory
+
+
+class InformationCategoryTests(APITestCase):
+    @classmethod
+    def setUpTestData(self):
+        super().setUpTestData()
+        self.information_category_value_list = InformationCategoryFactory.create(
+            identifier="https://www.example.com/waardenlijsten/1",
+            naam="first item",
+            naam_meervoud="first items",
+            definitie="This is some information about the first item.",
+            order=0,
+            oorsprong=InformationCategoryOrigins.value_list,
+        )
+        self.information_category_custom_entry = InformationCategoryFactory.create(
+            identifier="https://www.example.com/waardenlijsten/2",
+            naam="second item",
+            naam_meervoud="second items",
+            definitie="This is some information about the second item.",
+            order=1,
+            oorsprong=InformationCategoryOrigins.custom_entry,
+        )
+
+    def test_list_informatie_categorie(self):
+        list_url = reverse("api:informationcategory-list")
+        response = self.client.get(list_url)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.json()
+        self.assertEqual(data["count"], 2)
+
+        # First item
+        self.assertEqual(
+            data["results"][0]["identifier"], "https://www.example.com/waardenlijsten/1"
+        )
+        self.assertEqual(data["results"][0]["naam"], "first item")
+        self.assertEqual(data["results"][0]["naamMeervoud"], "first items")
+        self.assertEqual(
+            data["results"][0]["definitie"],
+            "This is some information about the first item.",
+        )
+        self.assertEqual(data["results"][0]["order"], 0)
+        self.assertEqual(
+            data["results"][0]["oorsprong"], InformationCategoryOrigins.value_list
+        )
```

Comment:
I prefer structuring these assertions in API tests as:

```py
expected = {
    "identifier": ...,
    "naam": ...,
    ...
}
self.assertEqual(data["results"][0], expected)
```

1. it's more readable what exactly is expected
2. if the assertion fails, the test itself will present a diff of which keys/values are not matching
3. it's easy to put a breakpoint for debugging to compare `expected` and `data["results"][0]` in a debug shell